tethys/DataReader/IniReader.py

```diff
@@ -25,11 +25,27 @@ def __init__(self, ini):
         self.InputFolder        = config['Project']['InputFolder']
         self.OutputFolder       = os.path.join(config['Project']['OutputFolder'], self.ProjectName)
         self.rgnmapdir          = config['Project']['rgnmapdir']
-        self.OutputFormat       = int(config['Project']['OutputFormat'])
-        self.OutputUnit         = int(config['Project']['OutputUnit'])
-        self.PerformDiagnostics = int(config['Project']['PerformDiagnostics'])
-        self.PerformTemporal    = int(config['Project']['PerformTemporal'])
-        
+        try: 

```

Comment:
Remove bare `except` statements.  We need to catch the actual error specifically.  For example: 
```python
except KeyError:
```